guide/yaml/winrm/index.md

```diff
@@ -271,6 +271,16 @@ config like `pre.install.reboot.required` and `install.reboot.required`. If requ
 installation commands can be split between the pre-install, install and post-install phases
 in order to do a reboot at the appropriate point of the VM setup.
 
+We Strongly recommend to **write blueprints in a way that they do NOT restart automatically windows**
+
+Using install commands that restart the Operating System automatically through a forked process
+will make the blueprint to behave inconsistently and report wrongly failure.
+
+To avoid such cases manually call the command `shutdown /r` or the powershell one `Restart-Computer`.
```

Comment:
See the config option "install.reboot.required". That handles reboot better than executing this command directly.